src/main/java/mezz/jei/config/Config.java

```diff
@@ -303,11 +303,24 @@ public static void preInit(FMLPreInitializationEvent event) {
 			}
 		}
 
+		bookmarkFile = new File("./", "jei_bookmarks.ini");

```

Comment:
```suggestion
		bookmarkFile = new File(Loader.instance().getConfigDir().getParent(), "jei_bookmarks.ini");

```
I think we should be careful to get the right directory here, I am not sure what weird setups people might have with unusual `cwd`s.